server/pages/index.jade

```diff
@@ -1,135 +0,0 @@
-doctype html
-//
```

Comment:
The doctype? `express-react-views` produces an HTML5 one by default, though, so we aren't really losing it.